src/Y79Y.yaml

```
@@ -0,0 +1,107 @@
+---
+- name: Tabs in various contexts
+  from: '@ingydotnet'
+  tags: whitespace
+  fail: true
+  yaml: |
+    foo: |
+    ————»
+    bar: 1
+  tree: |
+    +STR
+     +DOC
+      +MAP
+       =VAL :foo
+       =VAL |
+
+- yaml: |
+    foo: |
+     ———»
+    bar: 1
+  tree: |-
+    +STR
+     +DOC
+      +MAP
+       =VAL :foo
+       =VAL |\t\n
+       =VAL :bar
+       =VAL :1
+      -MAP
+     -DOC
+    -STR
+  json: |
+    {
+      "foo": "\t\n",
+      "bar": 1
+    }
+
+- yaml: |
+    - [
+    ————»
+     foo
+     ]
+  tree: |-
+    +STR
+     +DOC
+      +SEQ
+       +SEQ []
+        =VAL :foo
+       -SEQ
+      -SEQ
+     -DOC
+    -STR
+  json: |
+    [
+      [
+        "foo"
+      ]
+    ]
+
+- fail: true
+  yaml: |
+    - [
+    ————»foo,
+     foo
+     ]
+  tree: |
+    +STR
+     +DOC
+      +SEQ
+       +SEQ []
+
```

missing json:
```suggestion
  json: null
```
Maybe the geneartor should warn about that because in `fail` cases `json` doesn't make sense.

---

Since the `tree` of an error test is very implementation specific the new test generator just uses YAML::PP's events. So it's odd to me that you would say any of them are wrong.

I don't add json sections for any error tests.